app/views/shelters/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1> All Shelters </h1>
+
+  <% @shelters.each do |shelter| %>
+    <h2><%= shelter.name %></h2>
```

Comment:
- Created shelters instance variable in shelters_controller.rb to access all shelter instances.